### PR TITLE
use nanocosmos player insted of iframe

### LINKED ISSUE #2246: Nanocosmos stream always starts mute
We want the Nanocosmos stream to start automatically with sound.

Currently the stream is in an iframe so we have no control over the stream. Autostart is supported by Nanocosmos only mute ([Nanocosmos docs](https://docs.nanocosmos.de/docs/nanoplayer/nanoplayer_autoplay)).

We hope when we use their [NanoPlayer](https://docs.nanocosmos.de/docs/nanoplayer/nanoplayer_api) that we can then get control over the sound via the api and [unmute](https://docs.nanocosmos.de/docs/nanoplayer/nanoplayer_api/#nanoplayerunmute) the sound.

### LINKED ISSUE #2248: The livestream poster image is not displayed
If the livestream is not running, the image configured under 'Setting' -> 'Livestream' -> 'Livestream poster image url' should be displayed. Currently no image is displayed. 

Maybe it makes sense to do #2246 first.

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #2246 
resolves #2248

### CODE REVIEW

**Review by emanuelschuetze — APPROVED** [review]
Tested with Chromium only. Looks good.

**Comment by m-schieder** [comment]
I have tested this again. It seems that the live stream continues in the background when you enter the conference.
After you leave the conference room, a second stream is started. The old one continues to run in the background, which is why you can still hear the sound.

Tested in FF and on Android.

File: client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts [hunk 1]
```diff
@@ -56,19 +58,39 @@ export class VideoPlayerComponent implements AfterViewInit, OnDestroy {
...
             }
+        } else if (this.usingNanocosmos) {
+            this.videoId = this.getNanocosmosVideoId(this.videoUrl);
+
+            if (!this.nanoPlayer) {
+                const script = document.createElement(`script`);
+                script.type = `text/javascript`;
+                script.src = `https://demo.nanocosmos.de/nanoplayer/api/release/nanoplayer.4.min.js`;
+                script.onload = () => {
+                    // @ts-ignore
```
Comment (luisa-beerboom) [inline client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts:82]: Why are you using ts-ignore here?
At least add a comment as to why

File: client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts [hunk 2]
```diff
@@ -184,6 +208,45 @@ export class VideoPlayerComponent implements AfterViewInit, OnDestroy {
...
     }
 
+    public async updateNano(): Promise<void> {
+        let entry: any = {
+            bintu: {
+                streamid: this.videoId
+            }
+        };
+
+        // TODO: Including via rtmp streams is deprecated.
```
Comment (luisa-beerboom) [inline client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts:218]: Solve this now or create a cleanup issue, please.

**Review by luisa-beerboom — CHANGES_REQUESTED** [review]

File: client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts (hunk 1 above)
Comment (luisa-beerboom) [reply client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts:82]: Also: This function is way too long, would it be possible to move some of it to a helper function or two?

**Review by luisa-beerboom — CHANGES_REQUESTED** [review]

File: client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts (hunk 1 above)
Comment (bastianjoel) [reply client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts:82]: `NanoPlayer` is loaded from an external source and therefore not known to typescript. I don't think this needs explanation as this is already pretty clear by looking at the code above which embeds the external scripts.

File: client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts (hunk 2 above)
Comment (bastianjoel) [reply client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts:218]: This needs to be included for backward compatibility. I will create an issue.

**Review by luisa-beerboom — APPROVED** [review]

**Pull request merged**